README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# Weather App :sunny: :zap: :rainbow: :cloud_with_rain: :tornado:
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Description
 
-## Available Scripts
+A Weather App UI built using React, which can be used to view a 5 day weather forecast for a specified location, as well as a detailed forecast for any selected day.
 
-In the project directory, you can run:
+## Purpose
 
-### `npm start`
+To allow me to gain an understanding of React, building an app out of components, using state and event handlers and accessing external data in a React app.
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+## Tech that was used
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+- React
+- Axios
 
-### `npm test`
+## Why this tech?
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+I used React because it is scalable and renders quickly and Axios because it automatically transforms responses to JSON data;
```

Comment:
Dont need the semi colon at the end here
```
I used React because it is scalable and renders quickly and Axios because it automatically transforms responses to JSON data
```